### ISSUE TITLE
E731 lambda not detected without arguments or with default arguments

### ISSUE THREAD

**Issue by spaceone** [hhatto/autopep8#279]
```
printf 'foo = lambda: bar()\n' | ~/autopep8.py - | pep8 -
stdin:1:1: E731 do not assign a lambda expression, use a def

printf 'foo = lambda : bar()\n' | ~/autopep8.py - | pep8 -
stdin:1:1: E731 do not assign a lambda expression, use a def

printf 'foo = lambda :bar()\n' | ~/autopep8.py - | pep8 -
stdin:1:1: E731 do not assign a lambda expression, use a def
```

**Comment by spaceone** [comment]
```
foo = lambda key, default=None: bar.get('%s/%s' % (prefix, key), default)
```

**Comment by hhatto** [comment]
Thanks for reporting.

fix in version 1.3.

**Comment by spaceone** [comment]
I think you didn't fix the default argument thing, did you?:
`foo = lambda key, default=None: bar.get('%s/%s' % (prefix, key), default)`

**Comment by hhatto** [comment]
I see

**Comment by hhatto** [comment]
fix in latest master (ba30b1ff900d5834790fc0fef1b00236517a4deb)

Thanks.
